Sources/StreamChat/Database/DTOs/ThreadReadDTO.swift

```diff
@@ -63,4 +87,50 @@ extension NSManagedObjectContext {
         dto.unreadMessagesCount = Int64(payload.unreadMessagesCount)
         return dto
     }
+
+    func loadThreadRead(parentMessageId: MessageId, userId: String) -> ThreadReadDTO? {
+        ThreadReadDTO.load(parentMessageId: parentMessageId, userId: userId, context: self)
+    }
+
+    func loadThreadReads(for userId: UserId) -> [ThreadReadDTO] {
+        ThreadReadDTO.load(userId: userId, context: self)
+    }
+
+    func markThreadAsRead(parentMessageId: MessageId, userId: UserId, at: Date) {
```

Comment:
nit: `at date: Date`

Reply:
It was copy pasted from another function, but yes makes sense 👍